database/migrations/20190921201746_chore-tracker.js

```diff
@@ -0,0 +1,68 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('User', tbl => {
+        tbl.increments(); //Id's auto increment
+
+        tbl
+            .string('name')
+            .notNullable()
+        tbl
+            .string('username')
+            .unique()
+            .notNullable()
+        tbl
+            .string('email')
+        tbl
+            .string('password')
+    })
+
+    .createTable('Chore', tbl => {
+        tbl.increments(); //Id's auto increment
+
+        tbl
+            .string('name')
+            .notNullable()
+        tbl
+            .string('how_long')
```

Comment:
what is "how long" referring to? modify variable name to something more descriptive?